packages/grid/x-grid/src/XGrid.test.tsx

```diff
@@ -165,13 +164,12 @@ describe('<XGrid />', () => {
 
     const { container, setProps } = render(<TestCase width={300} />);
     let rect;
-
     rect = container.querySelector('[role="row"][data-rowindex="0"]').getBoundingClientRect();
     expect(rect.width).to.equal(300 - 2);
 
     setProps({ width: 400 });
-    await raf(); // wait for the AutoSize's dimension detection logic
```

Comment:
I would keep the explanation for why each await is required. It helps bring clarity.